app/src/main/res/layout/videoplayer_activity.xml

```diff
@@ -101,7 +101,7 @@
                 android:textStyle="bold" />
 
             <TextView
-                android:id="@+id/txtvLength"
+                android:id="@+id/lengthButton"
```

Comment:
Could you please call this `durationLabel` instead? Length is a bit strange, even though it was called length before. Also, it is mostly used to display text. Not many know that it can also be tapped.

Same for the video player.